process/block/poolscleaner/nilpoolscleaner.go

```diff
@@ -0,0 +1,28 @@
+package poolscleaner
+
+// NilPoolsCleaner will be used when an PoolsCleaner is required, but another one isn't necessary or available
+type NilPoolsCleaner struct {
```

Comment:
Might delete this struct or add tests for it

Reply:
Struct was deleted.